makepanda/makepanda.py

```diff
@@ -1213,7 +1213,7 @@ def CompileCxx(obj,src,opts):
                 cmd += " /Qopt-report:2 /Qopt-report-phase:hlo /Qopt-report-phase:hpo"    # some optimization reports
             else:
                 cmd += " /W1 "
-            cmd += " /EHa /Zm300 /DWIN32_VC /DWIN32"
+            cmd += " /EHa /Zm300 /D_WIN32"
```

Comment:
`/D_WIN32` doesn't need to be passed to the compiler, it's implicit. (The `-D_WIN32` passed to Interrogate is still needed, however)